Taggers/python/diphotonDumpConfig_cff.py

```diff
@@ -11,7 +11,7 @@
     categories = cms.VPSet(),
 
     workspaceName = cms.untracked.string("cms_hgg_$SQRTS"),
-    nameTemplate = cms.untracked.string("$PROCESS_$SQRTS_$LABEL_$SUBCAT_$SYST"),
+    nameTemplate = cms.untracked.string("$PROCESS_$SQRTS_$CLASSNAME_$SUBCAT_$SYST"),
```

Comment:
As discussed with @musella this is to be reverted to `$PROCESS_$SQRTS_$LABEL_$SUBCAT` and then nameTemplate will be changed to `$PROCESS_$SQRTS_$CLASSNAME_$SUBCAT_$SYST`  directly in the systematic dumping cmssw config.